linux_os/guide/system/accounts/accounts-physical/screen_locking/console_screen_locking/configure_tmux_lock_keybinding/tests/correct.pass.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# platform = Oracle Linux 8,Red Hat Enterprise Linux 8,multi_platform_fedora
```

Comment:
Why do the test scenarios contain Oracle Linux 8 if the rule doesn't have OL 8 in prodtype in rule.yml?

Reply:
Copy pasta error, :P